docs/users.rst

```diff
@@ -226,3 +226,60 @@ These tags are applied to the manifest list or, if multi-platform
 image builds are not enabled (see :ref:`group_manifests
 <group-manifests>`), to the sole image manifest resulting from the
 build.
+
+Overridde Parent Image
+----------------------
+
+The parent image used for building a layered image is determined by the ``FROM``
+instruction in the Dockerfile by default. Users can override this behavior
+by specifying a koji parent build via the ``koji_parent_build`` API parameter.
+When given, the image reference in the provided koji parent build will be used as
+the value of the FROM instruction. The same source registry restrictions
+apply.
+
+Additionally, the koji parent build must use the same container image repository
+as the value of the FROM instruction in Dockerfile. For instance, if the
+Dockerfile states ``FROM fedora:27``, the koji parent build has to be of a
+container image that pushed to the ``fedora`` repository. The koji parent build
+may refer to a ``fedora:26`` image, but using a koji parent buid for an image
+that was pushed to ``rsyslog`` will cause a build failure.
+
+This behavior requires koji integration to be enabled in the OSBS environment.
+
+Koji NVR
+--------
+
+When koji integration is enabled, every container image build requires a unique
+Name-Version-Release, NVR. The Name and Version are extracted from the **name**
+and **version** labels in Dockerfile. Users can also use the **release** label
+to hard code the release value. Although, this requires a git commit for every
```

Comment:
Suggest making this one sentence: "... hard code the release value, although this requires ...".